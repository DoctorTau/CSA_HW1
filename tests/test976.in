337
3068 2179 8654 2621 -4693 7455 9956 -1736 5596 -2640 8969 6064 7935 5512 -3790 -2925 6001 2406 5129 -2487 -7527 3744 9650 -9220 -2688 3782 1533 -2221 -697 -470 2253 -7945 -510 9360 7997 -1092 -1765 -6088 -1670 6709 -5201 2971 4113 -5783 -4651 8949 -9014 -5245 -4821 6702 4746 5373 7837 -2706 499 -2220 9805 -7105 -4850 -1473 3097 -2683 461 -1857 -7570 -120 -9313 -8784 1972 8404 1644 -9677 4502 -6957 -1508 -4601 -7350 -7074 7641 3164 1692 -4815 -5583 3957 -8999 -4966 -5284 -6072 867 -2212 -8605 -5150 -7318 6614 9311 5124 8723 -2145 6045 -1124 -2543 4147 4184 -3761 1842 -6076 -6381 8286 8471 -5095 -1793 3010 8528 96 3618 -2565 6536 4307 6491 2454 4495 6474 8614 2436 9919 1147 -9685 7042 -8273 9750 792 -1417 8300 4573 8263 5845 -2829 8960 7736 4236 -3689 -989 -4073 4888 -1633 6633 -2869 -424 -6096 8866 -3717 -5971 -1069 628 -4776 -5711 8971 -1896 -3901 -1351 -1344 890 2482 365 9930 4237 4706 5480 6436 -6370 -2841 4926 7082 4013 -7071 -8519 -4624 -121 7586 9879 -3774 -7389 -5986 -2230 7958 -8531 7027 1554 6773 18 -6516 -7201 -7563 334 -8064 -5426 -6545 1683 -3260 -316 -4025 6707 7847 -4158 8773 798 5857 -8649 -6015 8532 -2418 5280 -4502 8429 -3656 -6773 4028 6186 15 -903 7875 -9599 3983 -7477 -9678 9774 -276 9728 278 5148 184 7960 6803 7477 -4073 -1013 4527 -1517 6878 712 -7636 -3150 -1143 5220 5318 8275 -4875 611 -4332 7893 -6481 1835 2345 -3824 1625 -6365 6490 2920 -6765 3081 7167 -8730 2844 5179 -6394 -1220 -668 -4170 -3589 -1110 9247 -9289 387 -609 -3341 -7193 -4400 -639 -5068 -5272 -2800 -3958 971 -3607 -2487 1330 6627 -2178 7740 -6352 -2168 -570 944 -6821 9295 4983 5715 -2491 7235 -2210 -7998 -5907 6750 -4904 3399 6681 -5705 3117 -1960 8831 -6487 -9670 -4953 -9878 8190 -6331 -9386 9513 -1313 9898 -3206 -2079 623 -2817 -5902 -2128 1722 8043 1352 1135 -1627 9825 -8250 6493 5054 2829 -2263 