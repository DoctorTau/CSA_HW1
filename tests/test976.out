-1736 -2640 -3790 -9220 -2688 -470 -510 -1092 -6088 -1670 -9014 -2706 -2220 -4850 -7570 -120 -8784 -1508 -7350 -7074 -4966 -5284 -6072 -2212 -5150 -7318 -1124 -6076 -424 -6096 -4776 -1896 -1344 -6370 -4624 -3774 -5986 -2230 -6516 -8064 -5426 -3260 -316 -4158 -2418 -4502 -3656 -9678 -276 -7636 -3150 -4332 -3824 -8730 -6394 -1220 -668 -4170 -1110 -4400 -5068 -5272 -2800 -3958 -2178 -6352 -2168 -570 -2210 -7998 -4904 -1960 -9670 -9878 -9386 -3206 -5902 -2128 -8250 